tools/nl/embeddings/input/base/sheets_svs.csv

```diff
@@ -3602,3 +3602,5 @@ worldBank/4_1_SHARE_RE_IN_ELECTRICITY,Renewable electricity share of total elect
 worldBank/EG_ELC_ACCS_RU_ZS,percentage of rural population with access to electricity
 worldBank/EG_ELC_ACCS_UR_ZS,percentage of urban population with access to electricity
 worldBank/EG_ELC_ACCS_ZS,percentage of population with access to electricity
+sdg/SG_GEN_PARL.SEX--F,proportion of seats held by women in national parliaments
+sdg/SG_GEN_PARLN.SEX--F,number of seats held by women in national parliaments
```

Comment:
nit:  I wonder if we should skip number?  the pc denominator is not valid, and we have some risk of number coming in the way of proportion result.

Reply:
that makes sense, updated & no new diffs